src/rust/engine/src/context.rs

```diff
@@ -45,7 +45,7 @@ pub struct Core {
   pub intrinsics: Intrinsics,
   pub runtime: Runtime,
   pub executor: task_executor::Executor,
-  store: Store,
+  pub store: Store,
```

Comment:
I don't remember why we're inconsistent on this, but: there is an accessor method for this. Fine with exposing it though.